namerd/docs/storage.md

```diff
@@ -98,4 +98,5 @@ token | no auth | The auth token to use when making API calls.
 datacenter | uses agent's datacenter | The datacenter to forward requests to.
 readConsistencyMode | `default` | Select between [Consul API consistency modes](https://www.consul.io/docs/agent/http.html) such as `default`, `stale` and `consistent` for reads.
 writeConsistencyMode | `default` | Select between [Consul API consistency modes](https://www.consul.io/docs/agent/http.html) such as `default`, `stale` and `consistent` for writes.
+failFast | `false` | If `false`, disable fail fast and failure accrual for Consul client. Should be changed to `true` if `host` is a load-balanced DNS name
```

Comment:
The advice in the last sentence is a bit confusing.  Maybe say something about keeping it false if talking to consul via the local consul agent but setting to true if talking directly to a HA consul api server?